cli/tests/integration/watcher_tests.rs

```diff
@@ -427,9 +427,8 @@ fn run_watch_with_import_map_and_relative_paths() {
   temp_directory.close().unwrap();
 }
 
-// TODO(bartlomieju): flaky (https://github.com/denoland/deno/issues/10552)
-#[ignore]
 #[test]
```

Comment:
This should be removed

Reply:
Huh, this does not match my working tree... did something weird here.